### PR TITLE
Add `abnormallyHyperplasticAnatomicalEntity.yaml`

### LINKED ISSUE #789: Add new hyperplastic anatomical entity related pattern
1. [x] add new pattern `abnormallyHyperplasticAnatomicalEntity.yaml` for review
2. [x] get feedback
3. [x] fix or improve pattern based on feedback

* Examples:
    * [Thyroid hyperplasia](http://purl.obolibrary.org/obo/HP_0008249)
    * [mammary gland hyperplasia](http://purl.obolibrary.org/obo/MP_0000630)
    * [iris hyperplasia](http://purl.obolibrary.org/obo/MP_0013765)

### PR DESCRIPTION
**Pull request by rays22**
This commit intends to
add a new hyperplastic anatomical entity related DOSDP pattern.

If applied, this commit will fix #789.

### CODE REVIEW

File: src/patterns/dosdp-dev/abnormallyHyperplasticAnatomicalEntity.yaml [hunk 1]
```diff
@@ -0,0 +1,50 @@
+pattern_name: abnormallyHyperplasticAnatomicalEntity.yaml
+
+pattern_iri: http://purl.obolibrary.org/obo/oba/patterns/abnormallyHyperplasticAnatomicalEntity.yaml
```
Comment (matentzn) [inline src/patterns/dosdp-dev/abnormallyHyperplasticAnatomicalEntity.yaml:3]: oba..

File: src/patterns/dosdp-dev/abnormallyHyperplasticAnatomicalEntity.yaml [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+
+description: 'Hyperplastic anatomical entity, for example, thyroid gland hyperplasia.'
+
+examples:
+  - http://purl.obolibrary.org/obo/HP_0008249
+  - http://purl.obolibrary.org/obo/MP_0000630
+  - http://purl.obolibrary.org/obo/MP_0013765
```
Comment (matentzn) [inline src/patterns/dosdp-dev/abnormallyHyperplasticAnatomicalEntity.yaml:10]: love the examples!

File: src/patterns/dosdp-dev/abnormallyHyperplasticAnatomicalEntity.yaml [hunk 3]
```diff
@@ -0,0 +1,50 @@
+pattern_name: abnormallyHyperplasticAnatomicalEntity.yaml
+
+pattern_iri: http://purl.obolibrary.org/obo/oba/patterns/abnormallyHyperplasticAnatomicalEntity.yaml
+
+description: 'Hyperplastic anatomical entity, for example, thyroid gland hyperplasia.'
```
Comment (matentzn) [inline src/patterns/dosdp-dev/abnormallyHyperplasticAnatomicalEntity.yaml:5]: Some thing a bit more for those of us that do not know what "hyperplastic" means. The main goal of the description is to ensure that the ontology developer can decide if the pattern is appropropriate compared to other _similar_ patterns.

**Review by matentzn — COMMENTED** [review]
Well done!

**Comment by rays22** [comment]
@matentzn , do you have any suggestion of what/how to fix the problem that causes the QC check failure?

**Comment by sbello** [comment]
my ORCID is orcid.org/0000-0003-4606-0597

**Comment by ybradford** [comment]
is this where you want ORCiDs? 
0000-0002-9900-7880

**Pull request merged**